src/DIRAC/WorkloadManagementSystem/Service/TornadoSandboxStoreHandler.py

```diff
@@ -0,0 +1,23 @@
+""" Tornado-based HTTPs SandboxStore service.
+
+.. literalinclude:: ../ConfigTemplate.cfg
+  :start-after: ##BEGIN TornadoSandboxStore
```

Comment:
This should be added to ConfigTemplate.cfg

Reply:
Done